client/src/views/CellView.tsx

```diff
@@ -101,20 +122,62 @@ const CellView = () => {
     });
   }, [Alert, Confirm, brute, t]);
 
+  useEffect(
+
+    () => {
+      let touchstartX = 0;
+      let touchendX = 0;
```

Comment:
You need to check if the current brute has the same user as the current user at the start of your useEffect, and not do anything if not